src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -22,40 +22,6 @@ import scala.io.Source
 case class VCFHeaderInfo(sampleIds: Array[String], infoSignature: TStruct, vaSignature: TStruct, genotypeSignature: TStruct,
   filtersAttrs: VCFAttributes, infoAttrs: VCFAttributes, formatAttrs: VCFAttributes)
 
-object VCFLine {
-  def invalidRef(ref: String): Boolean = {
-    var i = 0
-    while (i < ref.length) {
-      val c = ref(i)
-      (c: @switch) match {
-        case 'A' | 'C' | 'G' | 'T' | 'N' |
-             'a' | 'c' | 'g' | 't' | 'n' =>
-        case _ =>
-          return true
-      }
-      i += 1
-    }
-    false
-  }
-
-  def symbolicAlt(alt: String): Boolean = {
```

Comment:
I've been looking forward to deleting this for a while.  That said, we should add alt allele support to the allele functions ASAP.

Reply:
I think I added most of it. I guess the one thing missing is a function that checks if a string is a valid allele?